Rivet/Functions/Initialize-Database.ps1

```diff
@@ -9,13 +9,27 @@ function Initialize-Database
     param(
         [Parameter(Mandatory=$true)]
         [Rivet.Configuration.Configuration]
-        $Configuration
+        $Configuration,
+
+        [Switch] $Pop

```

Comment:
Why did you add this `Pop` switch?

Reply:
Invoke-Rivet function calls Initialize-Database, so when testing I noticed that when I ran Invoke-Rivet -Pop it would hit Initialize-Database and it want's to reapply the schema.ps1 file but it has already been applied to the database so it throws an error. I also think we spoke about not popping the contents of the schema.ps1 file.

Reply:
Hmm. Update-Database should be handling and skipping applying the schema.ps1 file if it's already been applied. Looks like that's a bug in Update-Database.